environment.yml

```diff
@@ -2,17 +2,20 @@ name: spopt
 channels:
   - conda-forge
 dependencies: 
-  - python=3.8
-  - scipy>=1.0
-  - numpy>=1.3
-  - pandas>=1
-  - networkx
-  - libpysal
+  - python=3.9
   - geopandas>=0.7.0
+  - libpysal
   - matplotlib
+  - networkx
+  - numpy>=1.3
+  - pandas>=1
+  - pulp
   - scikit-learn>=0.22
+  - scipy>=1.0
+  # testing, etc.
```

Comment:
the environment.yml is what defines the dependencies for the conda package, so i'd prefer to leave the testing and dev requirements out of it so we dont force casual users to install them if they're using conda

Reply:
and docs

Reply:
Fair enough. I'll remove.

Reply:
resolved in 5a44d17